ERCS/erc-659.md

```diff
@@ -0,0 +1,554 @@
+---
+eip: 659
```

Comment:
```suggestion
eip: 7776
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.

Reply:
Thanks.